README.md

```diff
@@ -243,7 +244,15 @@ func main() {
 		c.FullPath() == "/user/:name/*action" // true
 	})
 
-	router.Run(":8080")
+  // This handler will add a new router for /user/groups.
```

Comment:
tabs not spaces

Reply:
89f4ae2dd3369a6ee503c593d6cb287d8ec36b85